ccpm/claude_template/scripts/pm/search.sh

```
@@ -0,0 +1,71 @@
+    echo "  No matches"
+  fi
+  echo ""
+fi
+
+# Search in Epics
+if [ -d ".claude/epics" ]; then
+  echo "EPICS:"
+  results=$(find .claude/epics -name "epic.md" -exec grep -l -i "$query" {} \; 2>/dev/null)
+  if [ -n "$results" ]; then
+    for file in $results; do
+      epic_name=$(basename $(dirname "$file"))
```

Shellcheck (0.10.0)

[warning] Quote this to prevent word splitting.

(SC2046)